tools/docker/README.md

```diff
@@ -75,11 +75,21 @@ You can set the following variables on a per-command basis (`PORT_WORDPRESS=8000
 Configurable settings are documented in the [`./tools/docker/default.env` file](https://github.com/Automattic/jetpack/blob/master/docker/default.env).
 Customizations should go into a `./tools/docker/.env` file you create, though, not in the `./docker/default.env` file.
```

Comment:
Unrelated to this PR, but
```suggestion
Customizations should go into a `./tools/docker/.env` file you create, though, not in the `./tools/docker/default.env` file.
```
Feel free to ignore for this PR if you want, but then let's make a separate PR to fix that.